### DISCUSSION TITLE
Type conversion for header params removed in Connexion 3?

### DISCUSSION THREAD

**Discussion by Dulinn** [spec-first/connexion#1930] [Q&A]
Hi!
I have a question regarding handling of header params. I see that with [this change](https://github.com/spec-first/connexion/pull/1627), the type conversion has been removed from ParameterValidator. I understand that apparently this code was sometimes executed redundantly and has therefore been moved to a single place in the URIParser. However, to me it looks like that by doing that, header params are no longer being type-converted. 

I'm currently migrating a project from Connexion 2 to Connexion 3, and a API call that used to work in v2 now fails with "'False' is not of type 'boolean'", and digging into the code, I found the change mentioned above. Am I missing something? If not, can you suggest the cleanest way to restore the behaviour for type conversion of header params when migrating to v3?

Thanks a lot in advance!

**Comment by chrisinmtown** [comment]
I realize this is a very late reply, and of course I'm not a maintainer. Would you consider extending this discussion with a SSCE to demonstrate the behavior, what happens in V2 vs. what happens in V3 for header parameters?
